API_GUIDE.md

```diff
@@ -99,8 +99,8 @@ for data, target in train_loader:
 This snippet highlights how easy it is to switch your model to run on XLA. The
 model definition, dataloader, optimizer and training loop can work on any device.
 The only XLA-specific code is a couple lines that acquire the XLA device and
-step the optimizer with a <b>barrier</b>. Calling
-`xm.optimizer_step(optimizer, barrier=True)` at the end of each training
+step the optimizer. Calling
```

Comment:
Instead of "step the optimizer", "mark the step".

Reply:
will do 👍